tox.ini

```diff
@@ -13,11 +13,39 @@ commands =
         --basetemp={envtmpdir} \
         --junit-xml=xunit-{envname}.xml \
         --cov=mopidy_musicbox_webclient --cov-report=term-missing \
-        {posargs}
+        {posargs:tests/}
 
 [testenv:flake8]
 deps =
     flake8
     flake8-import-order
 skip_install = true
-commands = flake8
+commands = flake8 {posargs:mopidy_musicbox_webclient}
+
+[testenv:eslint]
+whitelist_externals =
+    /bin/bash
+deps =
+    nodeenv
+skip_install = true
+commands =
+    python build_env.py
```

Comment:
...this could be replaced with ``nodeenv -p --prebuilt`` but would be much slower.